frontend/epfl-mini-card-deck/controller.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace EPFL\Plugins\Gutenberg\MiniCardDeck;
+use \EPFL\Plugins\Gutenberg\Lib\Utils;
+
+require_once(dirname(__FILE__).'/../lib/utils.php');
+
+
+/**
+ * Render a Mini Card panel block
+ */
+function epfl_mini_card_panel_block($attributes)
+{
+    $legend         = Utils::get_sanitized_attribute( $attributes, 'legend' );
+    $link           = Utils::get_sanitized_attribute( $attributes, 'link' );
+    $image_id       = Utils::get_sanitized_attribute( $attributes, 'imageId' );
+    $image_url      = Utils::get_sanitized_attribute( $attributes, 'imageUrl' );
+    $openLinkNewTab = Utils::get_sanitized_attribute( $attributes, 'openLinkNewTab' , false);
+
+    if(empty(trim($link))) return "";
+
+    $image_post = empty($image_id)? null : get_post($image_id) ;
+
+    ob_start();
+?>
+    <div class="card">
+        <a href="<?php echo $link ?>" class="card link-trapeze-horizontal" <?php if($openLinkNewTab): ?> target="_new" <?php endif; ?>>
```

Comment:
```suggestion
        <a href="<?php echo $link ?>" class="card link-trapeze-horizontal" <?php if($openLinkNewTab): ?> target="_blank" <?php endif; ?>>
```
Je ne connaissais pas ce "_new", mais on dirait que c'est mieux sans :
https://stackoverflow.com/a/4964223